docs/Training-ML-Agents.md

```diff
@@ -8,17 +8,17 @@ Use the Python program, `learn.py` to train your agents. This program can be fou
 
 For a broader overview of reinforcement learning, imitation learning and the ML-Agents training process, see [ML-Agents Overview](ML-Agents-Overview.md).
 
-## Training with Learn.py
+## Training with learn.py
 
-Use the Python `Learn.py` program to train agents. `Learn.py` supports training with [reinforcement learning](Background-Machine-Learning.md#reinforcement-learning), [curriculum learning](Training-Curriculum-Learning.md), and [behavioral cloning imitation learning](Training-Imitation-Learning.md).
+Use the Python `learn.py` program to train agents. `learn.py` supports training with [reinforcement learning](Background-Machine-Learning.md#reinforcement-learning), [curriculum learning](Training-Curriculum-Learning.md), and [behavioral cloning imitation learning](Training-Imitation-Learning.md).
 
-Run `Learn.py` from the command line to launch the training process. Use the command line patterns and the `trainer_config.yaml` file to control training options.
+Run `learn.py` from the command line to launch the training process. Use the command line patterns and the `trainer_config.yaml` file to control training options.
 
 The basic command for training is:
 
-    python3 learn.py <env_file_path> --run-id=<run-identifier> --train
```

Comment:
The reason I changed this <env_file_path> to <env_name> here is because we had a definition for <env_name> earlier on in the https://github.com/Unity-Technologies/ml-agents/blob/0ddb0354e0905037c07738804c854cc8697d9f36/docs/Basic-Guides.md and in https://github.com/Unity-Technologies/ml-agents/blob/0ddb0354e0905037c07738804c854cc8697d9f36/docs/Getting-Started-with-Balance-Ball.md. 

And the reason in those places we used <env_name> was due to the context in those places.

Reply:
Okay, we just need to be clear that the name has to include the path.

Reply:
Emmm...Maybe I can add something following this to indicate here it doesn't need to be a file name, and it can be a path name either.

Reply:
you can just say: "where <env_name> is the name (including path) of your Unity executable containing..."

Reply:
Fixed.